src/Emulator/Tabs/ConfigureTab/ConfigureTab.tsx

```diff
@@ -32,15 +31,21 @@ const scrollDeviceContainer = (ref: React.MutableRefObject<HTMLDivElement | null
   // ref.current.scrollIntoView() on its own does not scroll down enough
   if (ref.current) {
     setTimeout(() => {
-      if (ref.current)
-        ref.current.scrollIntoView();
+      if (ref.current){
+        // Convert to HTMLDivElement, because TypeScript complains that
+        // .scrollTop and .scrollHeight does not exist.
+        // Convert to unknown first, because TypeScript complains that
+        // 'null' and 'HTMLDivElement' do not sufficiently overlap.
+        const element = ref.current as unknown as HTMLDivElement;
+        element.scrollTop = element.scrollHeight;
```

Comment:
You don't actually need to cast to unknown here since we accept a ref of type `HTMLDivElement` as an argument

```suggestion
        const element = ref.current;
        element.scrollTop = element.scrollHeight;
```